metrics/ci_integrations/test/client/github_actions/mappers/run_conclusion_mapper_test.dart

```diff
@@ -0,0 +1,197 @@
+      () {
+        const conclusion = RunConclusionMapper.failure;
+        final runConclusion = runConclusionMapper.map(conclusion);
+
+        const expectedRunConclusion = RunConclusion.failure;
+
+        expect(runConclusion, equals(expectedRunConclusion));
+      },
+    );
+
+    test(
+      ".map() maps the neutral run conclusion to the RunConclusion.neutral",
+      () {
+        const conclusion = RunConclusionMapper.neutral;
+        final runConclusion = runConclusionMapper.map(conclusion);
+
+        const expectedRunConclusion = RunConclusion.neutral;
+
+        expect(runConclusion, equals(expectedRunConclusion));
+      },
+    );
+
+    test(
+      ".map() maps the cancelled run conclusion to the RunConclusion.cancelled",
+      () {
+        const conclusion = RunConclusionMapper.cancelled;
+        final runConclusion = runConclusionMapper.map(conclusion);
+
+        const expectedRunConclusion = RunConclusion.cancelled;
+
+        expect(runConclusion, equals(expectedRunConclusion));
+      },
+    );
+
+    test(
+      ".map() maps the skipped run conclusion to the RunConclusion.skipped",
+      () {
+        const conclusion = RunConclusionMapper.skipped;
+        final runConclusion = runConclusionMapper.map(conclusion);
+
+        const expectedRunConclusion = RunConclusion.skipped;
+
+        expect(runConclusion, equals(expectedRunConclusion));
+      },
+    );
+
+    test(
+      ".map() maps the timed out run conclusion to the RunConclusion.timedOut",
+      () {
+        const conclusion = RunConclusionMapper.timedOut;
+        final runConclusion = runConclusionMapper.map(conclusion);
+        const expectedRunConclusion = RunConclusion.timedOut;
+
+        expect(runConclusion, equals(expectedRunConclusion));
+      },
+    );
+
+    test(
+      ".map() maps the action required run conclusion to the RunConclusion.actionRequired",
+      () {
+        const conclusion = RunConclusionMapper.actionRequired;
+        final runConclusion = runConclusionMapper.map(conclusion);
+
+        const expectedRunConclusion = RunConclusion.actionRequired;
+
+        expect(runConclusion, equals(expectedRunConclusion));
+      },
+    );
+
+    test(
+      ".map() maps the not specified run conclusion to null",
+      () {
+        final runConclusion = runConclusionMapper.map("TEST");
+
+        expect(runConclusion, isNull);
+      },
+    );
+
+    test(
+      ".map() maps the null run conclusion to null",
+      () {
+        final runConclusion = runConclusionMapper.map(null);
+
+        expect(runConclusion, isNull);
+      },
+    );
+
+    test(
+      ".unmap() unmaps the RunConclusion.success to successful run conclusion value",
+      () {
+        final runStatus = runConclusionMapper.unmap(RunConclusion.success);
+
+        const expectedRunStatus = RunConclusionMapper.success;
+
+        expect(runStatus, equals(expectedRunStatus));
```

Comment:
```suggestion
        final conclusion = runConclusionMapper.unmap(RunConclusion.success);

        const expectedConclusion = RunConclusionMapper.success;

        expect(conclusion, equals(expectedConclusion));
```
Here and in the below tests.